spell/spell-crafter-mainnet-workflow.md

```diff
@@ -129,6 +129,10 @@ Repo: https://github.com/makerdao/spells-mainnet
   * [ ] Ensure good coverage (every spell action is tested)
   * [ ] Ensure every test function is declared as `public`
   * [ ] IF the test needs to run, it MUST NOT have the `skipped` modifier; OTHERWISE, it MUST have the `skipped` modifier
+  * Testing of modules initialised via the spell
```

Comment:
nit, I think this also can be marked with `IF` 
eg. 
```
- [ ] IF new modules are initialised via the spell, the test must include
  - [ ] Sanity checks of the constructor arguments
  - [ ] Sanity checks of all values added/updated by the spell function
  - [ ] End-to-end "happy path" interaction with the module
```

Reply:
Agree that IF statement makes sense here, thanks for suggesting